server/src/test/java/io/crate/integrationtests/SubSelectIntegrationTest.java

```diff
@@ -779,4 +780,39 @@ public void test_non_recursive_with_query() {
         assertThat(printedTable(response.rows()),
             is("4| Arthur\n"));
     }
+
+    @Test
+    public void test_exists_with_subquery_returns_result_if_subquery_has_rows() {
+        String stmt = "SELECT mountain, height FROM sys.summits WHERE EXISTS (SELECT 1) order by height desc limit 2";
+        execute(stmt);
+        assertThat(printedTable(response.rows())).isEqualTo(
+            "Mont Blanc| 4808\n" +
+            "Monte Rosa| 4634\n"
+        );
+        stmt = """
+            SELECT mountain FROM sys.summits
+            WHERE EXISTS (SELECT 1 FROM sys.cluster WHERE 1 = 2) order by height desc limit 2
+            """;
+        execute(stmt);
+        assertThat(response.rowCount()).isEqualTo(0L);
+
+        stmt = """
+            SELECT
+                table_name
+            FROM
+                information_schema.tables t
+            WHERE
+                EXISTS
+                (
+                 SELECT 1
```

Comment:
Can we also have a test that selects an actual column of the table? 
maybe here: `SELECT column_name`